conda/conda-releaselogs/uamqp.md

```diff
@@ -1,5 +1,11 @@
 # uAMQP for Python (conda)
 
+## 2024.03.01
+
+### Packages included
+
+- uamqp-1.6.5
```

Comment:
I'm still debugging the build failure. That aside, this should be `1.6.8` given the version defined below.